wekan/values.yaml

```diff
@@ -52,6 +52,11 @@ service:
 ##
 endpoint: wekan.local
 
+## Main URL (including http:// or https://) where your Wekan
+## instance is accessible
+##
+root_url: https://wekan.local
```

Comment:
Should this be like previous change?

```
{{ .Values.root_url | quote }}
```

Reply:
No. This is the place where `.Values.root_url` is from.

Or do I misunderstand you?